harness/determined/pytorch/dsat/_dsat_search_method.py

```diff
@@ -901,9 +905,9 @@ def get_trial_list_after_successful_run(
 
     def should_stop_lineage(self, trial: DSATTrial) -> bool:
         # General conditions
-        failed_on_min_mbs = trial.error and trial.mbs == trial.search_data.lo
+        failed_on_min_mbs = trial.error and trial.mbs <= trial.search_data.lo
```

Comment:
Curious, should the logic not be `trial.error_in_direct_history` rather than an error on this particular trial?

Reply:
I think either should work, but the above seems to get at what we want to check more directly: namely whether the trial that just ran the smallest possible batch size failed. 

Should never happen, but if you had the `trial. error_in_direct_history and trial.mbs == trial.search_data.lo` check, then would get `True` in a scenario where the smallest possible batch size succeeded and just some other trial in the lineage failed.